# Protein Classification
## Sai Manoj Tekumalla

### In this module, protein classification is performed. InterProScan is used for this purpose. Proteins can be classified into groups according to sequence or structural similarity. These groups often contain well characterised proteins whose function is known. Thus, when a novel protein is identified, its functional properties can be proposed based on the group to which it is predicted to belong[1]. InterPro is a database which integrates together predictive information about proteins' function from a number of partner resources, giving an overview of the families that a protein belongs to and the domains and sites it contains[2]. InterPro is a database which integrates together predictive information about proteins' function from a number of partner resources, giving an overview of the families that a protein belongs to and the domains and sites it contains. Users who have novel nucleotide or protein sequences that they wish to functionally characterise can use the software package InterProScan to run the scanning algorithms from the InterPro database in an integrated way. Sequences are submitted in FASTA format. Matches are then calculated against all of the required member database's signatures and the results are then output in a variety of formats.

## Citations
### [1] Why classify proteins?
### Embl-Ebi
### https://www.ebi.ac.uk/training/online/courses/protein-classification-intro-ebi-resources/protein-classification/why-classify-proteins/
### [2] ebi-pf-team/interproscan
### Ebi-Pf-Team
### https://github.com/ebi-pf-team/interproscan 
